docs/usage.md

```diff
@@ -8,24 +8,29 @@ There are two ways to use the generator:
 - [Generator library](#using-as-a-modulepackage)
 
 ## AsyncAPI CLI
+Generates whatever you want using templates compatible with AsyncAPI Generator.
 ```bash
-Usage: asyncapi generate fromTemplate <asyncapi> <template> [<options>]
+USAGE
+  $ asyncapi generate fromTemplate [ASYNCAPI] [TEMPLATE] [-h] [-d <value>] [-i] [--debug] [-n <value>] [-o <value>] [--force-write] [-w] [-p <value>] [--map-base-url <value>]
 
-- <asyncapi>: Local path or URL pointing to AsyncAPI document for example https://bit.ly/asyncapi
-- <template>: Name of the generator template like for example @asyncapi/html-template or https://github.com/asyncapi/html-template
+ARGUMENTS
+  ASYNCAPI  - Local path, url or context-name pointing to AsyncAPI file
```

Comment:
```suggestion
  ASYNCAPI  - Local path, URL pointing to AsyncAPI document  for example https://bit.ly/asyncapi
```

Reply:
Hi @Florence-Njeri , @derberg 
I did "construct" my update by copy/pasting output of `asyncapi generate fromTemplate --help`. The suggestions break alignment between code and documentation. Do you really want to go that direction?

Alternative: apply suggestions to code (help text) and documentation in parallel. @derberg, can you point me to the help text?

Reply:
@Florence-Njeri normally I like that you want to clarify help with examples, like example link. Problem is that https://bit.ly/asyncapi is no longer the best as it points to asyncapi `2.0.0` and nobody knows who owns this short link and how to update it. So we should not promote it

so correct help is
`Local path, name of CLI context or URL pointing to AsyncAPI document`

Reply:
@eelcofolkertsma we can accept the update here now and if you could followup with PR to fix in in CLI in https://github.com/asyncapi/cli/blob/master/src/commands/generate/fromTemplate.ts#L106 that would be awesome